ditto/writers/opendss/write.py

```diff
@@ -135,9 +136,12 @@ def float_to_str(self, f):
         d1 = ctx.create_decimal(repr(f))
         return format(d1, "f")
 
-    def write(self, model, **kwargs):
+    def write(self, model, write_taps=False, **kwargs):
```

Comment:
it's design choice; I find it much better then catching everything in kwargs because then one can tell what options are available via the function signature (rather then having to read through the code logic)

Reply:
also note that this change turns four lines of code below into one
```python
        if "write_taps" in kwargs:
            self.write_taps = kwargs["write_taps"]
        else:
            self.write_taps = False
```
becomes
```python
        self.write_taps = write_taps
```

Reply:
ok. I'll do this for the other kwargs.